### PR TITLE
CNFT2-2268 Library icon tooltips

### PR DESCRIPTION
**Pull request by jlee-enquizit**
## Description

Adding tooltips to Page Library and Business Rules libraries
image
image

## Tickets

* [CNFT2-2268](https://cdc-nbs.atlassian.net/browse/CNFT2-2268)

## Checklist before requesting a review
- [x] PR focuses on a single story
- [x] Code has been fully tested to meet acceptance criteria
- [x] PR is reasonably small and reviewable (Generally less than 10 files and 500 changed lines)
- [x] All new functions/classes/components reasonably small
- [x] Functions/classes/components focused on one responsibility
- [x] Code easy to understand and modify (clarity over concise/clever)
- [x] PRs containing TypeScript follow the [Do's and Don'ts](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html)
- [x] PR does not contain hardcoded values (Uses constants)
- [x] All code is covered by unit or feature tests

[CNFT2-2268]: https://cdc-nbs.atlassian.net/browse/CNFT2-2268?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by lramakr — DISMISSED** [review]

File: apps/modernization-ui/src/apps/page-builder/page/library/menu/PageLibraryMenu.tsx [hunk 1]
```diff
@@ -53,26 +53,32 @@ const PageLibraryMenu = ({ properties, filters, onSearch, onFilter, onDownloadCs
...
+                    <Button
+                        aria-label="download search results as csv"
+                        type="button"
+                        className={styles.icon}
+                        outline
+                        onClick={onDownloadCsv}
+                        data-testid="file-download">
+                        <Icon.FileDownload size={3} />
+                    </Button>
+                </Tooltip>
```
Comment (mpeels) [inline apps/modernization-ui/src/apps/page-builder/page/library/menu/PageLibraryMenu.tsx:81]: Another option is to use the css only tooltips provided by: https://github.com/CDCgov/NEDSS-Modernization/pull/1035

File: apps/modernization-ui/src/apps/page-builder/page/library/menu/page-library-menu.module.scss [hunk 2]
```diff
@@ -37,4 +37,12 @@
...
         }
     }
+    // .usa-tooltip {
+    //     // removing the button styling inserted by tooltip
+    //     .usa-button {
+    //         &.usa-tooltip__trigger {
+    //             background-color: white;
+    //         }
+    //     }
+    // }
```
Comment (mpeels) [inline apps/modernization-ui/src/apps/page-builder/page/library/menu/page-library-menu.module.scss:47]: Issue: commented out code

File: apps/modernization-ui/src/apps/page-builder/page/management/menu/page-management-menu.module.scss [hunk 3]
```diff
@@ -17,4 +17,5 @@
         display: flex;
         align-items: center;
     }
+    // gap: 0.25rem;
```
Comment (mpeels) [inline apps/modernization-ui/src/apps/page-builder/page/management/menu/page-management-menu.module.scss:20]: Issue: commented out code

File: apps/modernization-ui/src/styles/_uswds.scss [hunk 4]
```diff
@@ -108,11 +108,30 @@ table.usa-table {
...
+        margin: 0;
         &.usa-tooltip__trigger {
             background-color: unset;
             color: colors.$primary;
             padding: 0;
+                background-color: colors.$base-white;
+                a {
+                    box-shadow: inset 0 0 0 2px #005ea2;
+                    padding: 0.5rem 0.75rem;
+                }
```
Comment (mpeels) [inline apps/modernization-ui/src/styles/_uswds.scss:134]: Suggestion: All of this becomes unnecessary if we use the css only tooltips.

**Review by mpeels — CHANGES_REQUESTED** [review]

File: apps/modernization-ui/src/apps/page-builder/page/library/menu/PageLibraryMenu.tsx (hunk 1 above)
Comment (jlee-enquizit) [reply apps/modernization-ui/src/apps/page-builder/page/library/menu/PageLibraryMenu.tsx]: Implemented for Buttons.
But this method does not work with `LinkButton`

**Review by mpeels — APPROVED** [review]

**Review by alaapbharadwaj — APPROVED** [review]

**Pull request merged**